embedded-files/tekton/triggers.yaml

```diff
@@ -214,11 +234,28 @@ spec:
         secretKeyRef:
           name: mlflow-minio
           key: secretkey
-  - name: run-id-to-results
+  - name: model-uri-to-results
```

Comment:
does it make sense to have this in separate step? I think model-uri result could be already produced in the run step, is there a reason to start a new container?

Reply:
I have seen this pattern used in tekton and I decided to follow it. I think it makes sense as each step should have its own goal and in case of failure its easier to debug (e.g. the error was getting the URI or training the model?)